lib/topological_inventory/ansible_tower/collector.rb

```diff
@@ -86,8 +88,9 @@ def collector_thread(connection, entity_type)
       # connection.api.jobs.all
       # connection.api.workflow_jobs.all
     rescue => e
+      metrics.record_error
```

Comment:
@carbonin It should be probably there in all collectors, right?

Reply:
Are any of the collectors missing this?

Reply:
all of them. They are record errors only for targeted refresh, not in full_refresh's thread

Reply:
Okay, then make the change. My goal was to just make the metrics available in all the components then the people who are most familiar with the source can add whatever metrics they think would be useful.

Also, since you mentioned threads, I'm not sure the metrics object is thread safe. Maybe making it thread safe would be a candidate for a separate gem. cc @agrare

Reply:
Or maybe we can contribute that back to prometheus exporter itself